### PR TITLE
[docs]: add usage section in README

### LINKED ISSUE #135: Android - Complete Usage section
Complete the Usage section in the Android project README at `coding-projects/android/README.md`

### PR DESCRIPTION
**Pull request by CBID2**
## TODO:
Closes #135 
## TODO: Request a maintainer to review your PR
@ceceliacreates, My PR is ready for review! :)  

## TODO: Write a brief description of what your PR accomplished
This PR adds information about how users should interact with the to do list app. This helps contributors gain a better understanding of how to work on this project. 

## TODO: Attach screenshots of your changes
screenshot of usage section

## TODO: Explain how to test your PR
Click on the Markdown preview in VSCode to view the section in Markdown.

### CODE REVIEW

File: coding-projects/README.md [hunk 1]
```diff
@@ -14,26 +15,31 @@ Please follow the instructions below to get started.
...
   Here is a short summary of contribution steps:
-   - Fork this repository.
-   - Create a new branch: `git checkout -b feature/your-feature`.
-   - Commit your changes: `git commit -am 'Add new feature'`.
-   - Push to the branch: `git push origin feature/your-feature`.
-   - Submit a pull request.
+
+- Fork this repository.
+- Create a new branch: `git checkout -b feature/your-feature`.
+- Commit your changes: `git commit -am 'Add new feature'`.
```
Comment (clc80) [inline coding-projects/README.md:30]:
```suggestion
- Commit your changes: `git commit -m 'Add new feature'`.
```

There shouldn't be an "a" there. I know you didn't add that there but it really should be deleted.

**Review by clc80 — CHANGES_REQUESTED** [review]
Found a typo that could cause issues for beginners.

File: coding-projects/README.md (hunk 1 above)
Comment (CBID2) [reply coding-projects/README.md:30]: Hi @clc80, commit am is an actual Git command: https://git-scm.com/docs/git-am/2.33.1

Comment (clc80) [reply coding-projects/README.md:30]: @CBID2 I did not know that, but that is not the intended use of it in this case. There is no patching going on. I believe they meant to use -m to commit with a message as is the most common scenario.

Comment (CBID2) [reply coding-projects/README.md:30]: Hey @clc80. I realized that the article I sent you doesn't best explain this command. Here is a better one: https://teamtreehouse.com/community/git-commit-a-m-new-message-commits-all-changes-so-what-happens-when-i-just-type-git-commit-m-new-message

Comment (clc80) [reply coding-projects/README.md:30]: @CBID2 The article explains that this is for two separate commands done together. `-a ` and `-m` `git commit `-a -m` not `git commit -am` as is in the PR. So can it be fixed to one or the other and not left as `am` as this is not the one we want. 

if we use `-a -m` we need to add that we are using the -a to all al files.

Comment (CBID2) [reply coding-projects/README.md:30]: Ok

Comment (CBID2) [reply coding-projects/README.md:30]: Hi @clc80! :) Here's an article that explains this command better than the other sources I mentioned (scroll to the end to find it): https://modulesunraveled.com/very-basics-git/performing-commits-more-quickly-m-and-options

**Review by clc80 — APPROVED** [review]
Thanks for your work.

**Review by devanshimodha — APPROVED** [review]

**Pull request merged**